week-1/2-mandatory/3-function-output.js

```diff
@@ -1,24 +1,23 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/******************************************
+ * This function return a positive
+ * random number less than 9 including 0
+ * ****************************************/
 function getNumber() {
   return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+/******************************************
+ * This function takes two strings and
+ * join (concatenate) them together
+ * ****************************************/
 function s(w1, w2) {
   return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together
-  // Look at the test case below to understand what to expect in return
+  return firstWord + " ".concat(secondWord + " ", thirdWord);
```

Comment:
Nice. You could also use template strings to get the same result.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
```
return `${firstWord} ${secondWord} ${thirdWord}`
```